lib/processors/bundlers/flexChangesBundler.js

```diff
@@ -86,19 +88,41 @@ export default function({resources, options: {pathPrefix, hasFlexBundleVersion}}
 			return JSON.stringify(changes);
 		} else {
 			bundleName = "flexibility-bundle.json";
-			const newChangeFormat = {
+			let newChangeFormat = {
 				changes,
 				compVariants,
 				variants,
 				variantChanges,
 				variantDependentControlChanges,
 				variantManagementChanges
 			};
-
+			if (Object.keys(existingFlexBundle).length > 0) {
+				newChangeFormat = mergeFlexChangeBundles(newChangeFormat, existingFlexBundle);
+			}
 			return JSON.stringify(newChangeFormat);
 		}
 	}
 
+	/**
+	 * merge new and existing bundles
```

Comment:
style: I would recommend to use uppercase "M" when starting the jsdoc description

Reply:
done